portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -98,14 +101,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   /** Posts a comment retrieved from the form input adding it to the messages variable. */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+    
     // Get the comment and image from the form and add it to the array.
```

Comment:
Maybe update comment since you're now also retrieving the username?

Reply:
Done!